test/service/test_response.py

```diff
@@ -48,18 +48,16 @@ def project_value(v):
                         "count": 1,
                     }
                 ],
-                "processes": [
-                    {
-                        "_type": ["navy"],
-                        "document_id": ["4188ddbe-8865-a433-a1f6-213d49aa5719"],
-                        "instrument_manufacturer_model": ["green"],
-                        "library_construction_approach": ["fuchsia"],
-                        "process_id": ["maroon"],
-                        "process_name": ["olive"],
-                        "protocol_name": ["olive"],
-                        "protocol_id": ["green"]
-                    }
-                ],
+                "protocols": [
```

Comment:
The indention is wrong. Also, could you keep the singleton lists on one line so it's consistent with the other singletons in these literals?